server/routes/termdb.termsbyids.ts

```diff
@@ -59,5 +72,5 @@ async function trigger_gettermsbyid(
 	}
 	res.send({
 		terms
-	})
+	} as gettermsbyidsResponse)
```

Comment:
As a general rule, all typescript types, interfaces, and classes are in Pascal Case.